medusa/storage/s3_compat_storage/concurrent.py

```diff
@@ -26,7 +26,7 @@
 import medusa
 from medusa.storage.s3_compat_storage.awscli import AwsCli
 
-MAX_UP_DOWN_LOAD_RETRIES = 5
+MAX_UP_DOWN_LOAD_RETRIES = 10
```

Comment:
Did you see this change making a difference?
Maybe we should rather add a wait multiplier instead of performing fixed retries and then move back to 5 attempts?
wdyt?